source/_integrations/firmata.markdown

```diff
@@ -142,9 +142,9 @@ binary_sensor:
       required: true
       type: string
     pin:
-      description: The digital or analog pin number on the board.
+      description: The digital or analog pin number on the board. For analog pins the corresponding digital number must be used. Please use the pinout that corresponds to your board. For example, on the Arduino Uno, pin A2 corresponds to the digital pin number 16.
       required: true
-      type: [integer, string]
+      type: integer
```

Comment:
This is unfortunately incorrect/not true, the integration accepts a string.

See: https://github.com/home-assistant/core/blob/5fa6ef2849a9412e97526c98187ea68ed8b204f8/homeassistant/components/firmata/__init__.py#L81

And: <https://github.com/home-assistant/core/blob/5fa6ef2849a9412e97526c98187ea68ed8b204f8/homeassistant/components/firmata/__init__.py#L50>

Reply:
Ok, I reverted this commit.

Reply:
Yes, it *should* accept the form `A#` for pins (and that worked originally). This pull documents a less-intuitive workaround, not the intended method. See https://github.com/home-assistant/core/issues/63371